more/free-programming-cheatsheets.md

```diff
@@ -284,6 +285,11 @@
 * [Quick Cheat Sheet for Mongo DB Shell commands](https://gist.github.com/michaeltreat/d3bdc989b54cff969df86484e091fd0c) - Michael Treat's Quick Cheat Sheet (GitHub Gist)
 
 
+### Octave
+
+* [Octave Cheat Sheet](https://www.lehman.edu/academics/cmacs/documents/refcard-a4.pdf) (PDF)
```

Comment:
```suggestion
* [Octave Cheat Sheet](https://www.lehman.edu/academics/cmacs/documents/refcard-a4.pdf) -  John
W. Eaton (PDF)
```
Please add author names where appropriate.

Reply:
Oh thanks for this suggestion @eshellman , done the suggested changes :)